tilequeue/command.py

```diff
@@ -1074,6 +1074,19 @@ def tilequeue_prune_tiles_of_interest(cfg, peripherals):
                     coord_marshall_int(deserialize_coord(l.strip()))
                     for l in f
                 )
+        elif 's3' in info:
+            from boto import connect_s3
+            from boto.s3.bucket import Bucket
+            s3_conn = connect_s3()
+            bucket = Bucket(s3_conn, info['s3']['bucket'])
+            key = bucket.get_key(info['s3']['key'])
+            raw_coord_data = key.get_contents_as_string()
+            for line in raw_coord_data.splitlines():
+                coord = deserialize_coord(line.strip())
+                if coord:
+                    coord_int = coord_marshall_int(coord)
+                    uplifted_coord_int = coord_int_zoom_up(coord_int)
```

Comment:
Hmm, it might make sense to do the uplifting before it gets here. I agree that it's weird that this is the only path to do it. Maybe we can document somewhere that these "immortal tile" files/inputs are all assumed to be uplifted.